src/Window/sfml-window.csproj

```diff
@@ -6,7 +6,6 @@
 		<AssemblyName>sfml-window</AssemblyName>
 		<Configurations>Debug;Release;_WINDOWS_;_LINUX_;_OSX_</Configurations>
     <Description>Window module of the SFML library</Description>
-    <Version>2.4.2</Version>
```

Comment:
No replacement needed?

Reply:
Not sure why this isn't marking as Outdated